src/chainparams.cpp

```diff
@@ -79,10 +85,14 @@ class CMainParams : public CChainParams {
         consensus.BIP34Hash = uint256S("0x000000000000024b89b42a942fe0d9fea3bb44ab7bd1b19115dd6a759c0808b8");
         consensus.BIP65Height = 388381; // 000000000000000004c2b624ed5d7756c508d90fd0da2c7c679febfa6c4735f0
         consensus.BIP66Height = 363725; // 00000000000000000379eaa19dce8c9b722d46ae6a57c2f1a988119488b50931
-        consensus.BTGHeight = 487427; // Around 10/1/2017 12:00 UTC
-        consensus.BTGPremineWindow = 16000;
+        consensus.BTGHeight = 491407; // Around 10/25/2017 12:00 UTC
+        consensus.BTGPremineWindow = 8000;
         consensus.powLimit = uint256S("00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
-        consensus.nPowTargetTimespan = 14 * 24 * 60 * 60; // two weeks
```

Comment:
I’d rather rename nPowTargetTimespan to nPowTargetTimespanLegacy. And reuse nPowTargetSpacing.